source/adapters/hip/device.cpp

```diff
@@ -223,7 +224,19 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
     return ReturnValue(uint64_t{MaxAlloc});
   }
   case UR_DEVICE_INFO_IMAGE_SUPPORTED: {
-    return ReturnValue(ur_bool_t{true});
+    bool Enabled = false;
+
+    if (std::getenv("UR_HIP_ENABLE_IMAGE_SUPPORT") != nullptr) {
```

Comment:
I note that CUDA also has `SYCL_PI_CUDA_ENABLE_IMAGE_SUPPORT`. Do we need to mirror that?

Reply:
`SYCL_PI_CUDA_ENABLE_IMAGE_SUPPORT` is a leftover from the PI interface that pre-dates UR, it's in the CUDA adapter for legacy reasons, since we're introducing the variable for HIP I think it makes sense to just have the UR variant.

Reply:
Fair enough, thanks